### ISSUE TITLE
Idea: Default Equipment

### ISSUE THREAD

**Issue by uk3** [Runalyze/Runalyze#1795]
Hello,

I like the 'default Type' feature. And I think it would be useful to also have a Default for every equipment type (like i usually use a specific bike or specific shoes, and only sometimes another). Default would simplify the import for that 

lG Ulrich,

**Comment by mipapo** [comment]
I thought about this, but:
It would go to the equipment configuration. Every equipment configuration could have a default equipment object. 
What about multiple equipment categories? (There are mulitple checkboxes)
What about different sports like (Road bike, Mountainbike)? You may have one category for your bikes. (Would be resolved if we use the alternative idea)
-
Alternative: Only a default equipment can be selected for the main equipment per sport. (This would be the easiest)

**Comment by uk3** [comment]
Hello,

> On 06 May 2016, at 07:24, Michael Pohl <notifications@github.com> wrote:
> 
> I thought about this, but:
> It would go to the equipment configuration. Every equipment configuration could have a default equipment object. 
> What about multiple equipment categories? (There are mulitple checkboxes)
> 
> What about different sports like (Road bike, Mountainbike)? You may have one category for your bikes. (Would be resolved if we use the alternative idea)
> 
> Alternative: Only a default equipment can be selected for the main equipment per sport. (This would be the easiest)
> 
For my use-cases this would be the more useful approach.. 

lG Ulrich.

-- 
Ulrich Kiermayr, Vienna, AT
mail & jabber: ulrich@kiermayr.at <mailto:ulrich@kiermayr.at>

**Comment by uk3** [comment]
Hello,
Since I did an Upload of multiple activities, the issue popped up again for me.
For me it would be sufficient to have a default equipment for the main equipment per Sport. (like the default training type).
lG Ulrich

**Comment by laufhannes** [comment]
Is it enough to select *one* default equipment? This makes sense for me especially for cycling to select the default bike and maybe for running to select the default shoe. These are typically categories where only one equipment object can be chosen. I guess it's easy to add this column to the sport entity, but this will let users to select only one default equipment for categories that allow multiple objects to be chosen (e.g. clothes).

**Comment by uk3** [comment]
Yes exactly those are the cases I had in mind. Default shoes, default bike (and the default Bow for Archery ;-) )
